js/jasmine/spec/TowerOfHanoiSpec.js

```diff
@@ -0,0 +1,49 @@
+describe('Tower of Hanoi', () => {
+    const towerSourceName = "sourcePeg";
+    const towerTargetName = "auxPeg";
```

Comment:
warum hast du die als Klassenvariablen gemacht statt die in jedem Test neu zu instanziieren? Riecht ein bischen gefährlich, falls du am Ende von nem Test mal nicht richtig aufräumst

Reply:
Tatsächlich können die Konstanten weg, die brauche ich nicht mehr.  Hatte meinen Linter nicht initialisiert, sonst hätte der das angezeigt.

Reply:
Kannst du die bei der Übernahme löschen oder soll ich den Branch aktualisieren und einen neuen Request machen?

Reply:
ne ich merge ja nur die commits von dir in meinen master rein. Ich müsste auf deinen branch was committen, dafür bräuchte ich erstmal rechte etc... also mach einfach und push hier nochmal rein (kannst du einfach als "fix review comments" commit machen, ich kann die dann squashen vorm mergen)

Reply:
Also bzgl. Klassenvariablen habe ich jetzt auf einen "this." approach umgeändert (siehe https://gist.github.com/traviskaufman/11131303). Init und Rest habe ich dann in den Code an sich übernommen.
Strenggenommen waren das aber nur Variablen in meiner Suite, da im Scope von describe und m.E. absolut akzeptabel lt. Jasmine Intro :) Siehe SetUp und Teardown https://jasmine.github.io/2.0/introduction.html

Reply:
jo macht sinn - das ist genau das Problem das ich meinte. Hast du das schon gepusht?

Reply:
Hab neu gepusht. Mit einer weiteren Spec für Recursion (biggest disk never on smaller disk). Rebase habe ich aber nicht mehr gemacht

Reply:
So, rebase -i hat auch funktioniert meine ich. Kannst du das einmal checken, ob die history nun so aussieht wie erwartet? Und ja, ich musste push --force machen :)